mytry.js

```diff
@@ -0,0 +1,59 @@
+const mongoose = require("mongoose");
+
+// Adres URL bazy danych MongoDB
+// const { URI_DB: dbURI } = process.env;
+const dbURI =
+  "mongodb+srv://tomcekala:0vS4ptfWXj1TG3LT@cluster0.gjg16ja.mongodb.net/";
```

Comment:
To sobie przenieś do .env i dodaj ten plik do .gitignore żeby nie został wysłany na serwer. To powinny być dane prywatne. Następnie zrób .env.example gdzie napiszesz jakie zmienne stałe prywatne są potrzebne w tym repo np 

```
DB_URL=
PORT=
```

Reply:
Swoją drogą to ten plik wydaje się testowy, więc powinien być trzymany lokalnie a jeżeli już na repo to nie mergowany do mastera/main tylko trzymany na innej branch.